main.py

```
@@ -244,32 +244,69 @@ async def handler(data):
+{C_CYAN}║{C_RESET}  {C_GREEN}mine{C_RESET}                    - mine a block                  {C_CYAN}║{C_RESET}
+{C_CYAN}║{C_RESET}  {C_GREEN}peers{C_RESET}                   - show connected peers          {C_CYAN}║{C_RESET}
+{C_CYAN}║{C_RESET}  {C_GREEN}connect <multiaddr>{C_RESET}     - connect to a peer             {C_CYAN}║{C_RESET}
+{C_CYAN}║{C_RESET}  {C_GREEN}address{C_RESET}                 - show your public key          {C_CYAN}║{C_RESET}
+{C_CYAN}║{C_RESET}  {C_GREEN}chain{C_RESET}                   - show chain summary            {C_CYAN}║{C_RESET}
+{C_CYAN}║{C_RESET}  {C_GREEN}help{C_RESET}                    - show this help                {C_CYAN}║{C_RESET}
+{C_CYAN}║{C_RESET}  {C_GREEN}quit{C_RESET}                    - shut down                     {C_CYAN}║{C_RESET}
+{C_CYAN}╚══════════════════════════════════════════════════════════╝{C_RESET}
 """
 
 
 async def cli_loop(sk, pk, chain, mempool, network):
```

Ruff (0.15.18)

[warning] Too many branches (41 > 12)

(PLR0912)